src/mbgl/map/transform.cpp

```diff
@@ -46,6 +46,10 @@ Transform::Transform(MapObserver& observer_,
 #pragma mark - Map View
 
 bool Transform::resize(const Size size) {
+    if (!size) {
+        return false;
```

Comment:
If (0, 0) is an invalid size, shouldn't this throw rather than silently doing nothing?